### DISCUSSION TITLE
Error when handling "unknown" jobs on LSF system

### DISCUSSION THREAD

**Discussion by Ding-Hairui** [deepmodeling/dpdispatcher#211] [Q&A] [answered]
Dear maintainers, here is an error when running DPGEN:

![1](https://user-images.githubusercontent.com/47772564/182366626-94205b1e-19d0-48c3-9b0f-b8a28788ef55.png)

  Our machine uses LSF system to manage jobs. Sometimes the job state would become "UNKNOWN"* due to network's problems between computer nodes, and it would automatically return to normal ("RUN" or "PEND"). According to the error information, it seems that DPGEN ends running when meeting unknown jobs.
  If so, I would like to know whether there is any method to skip or ignore such situation, or could you please add some functions to process it? If not, I hope you can tell me why and how to deal with it. Thank you!

  DPGEN version:  0.10.7.dev0+g8dea29e.d20220801

*or maybe "UNKNWN" when querying job state. I hardly see it, and the LSF documentation does not mention it.

**Comment by njzjz** [comment, marked as answer]
https://github.com/deepmodeling/dpdispatcher/blob/a3773f9574a76d66d5e275dda4565b4d4fe49e3c/dpdispatcher/lsf.py#L135-L146

This is the current list of keywords to be processed. Contribution is welcome if you found new words.

**Reply by Ding-Hairui** [reply]
I found a similar issue at other project.

https://github.com/Snakemake-Profiles/lsf/issues/28

As it said, "UNKNWN" job is caused by kinds of problems on different cluster systems. It's difficult to generalize all situations. I will firstly fix it on our own machine, and then consider how to deal with it more gracefully. You can alse take that issue as reference to improve it. Thanks again.
